Applications/DataExplorer/DataView/ProcessView.h

```diff
@@ -0,0 +1,72 @@
+/**
+ * \file
+ * \author Karsten Rink
+ * \date   2010-12-13
+ * \brief  Definition of the ProcessView class.
+ *
+ * \copyright
+ * Copyright (c) 2013, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef PROCESSVIEW_H
+#define PROCESSVIEW_H
+
+#include <QContextMenuEvent>
+#include <QTreeView>
+
+#include "Applications/DataHolderLib/FemCondition.h"
+
+class ConditionModel;
+
+/**
+ * \brief A view for FEM-Conditions (Initial- & Boundary Conditions / Source
+ * Terms) with a number of additional information such as Process Type,
+ * Distribution, etc. \sa ConditionModel, CondItem
+ */
+class ProcessView : public QTreeView
```

Comment:
classes, from which a user is not supposed to derive another class, should be declared with `final` specifier.